pkg/util/uint160_test.go

```diff
@@ -5,75 +5,69 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
why not to use require everywhere instead of assert?

Reply:
Sometimes you want to see the result of several checks before failing the test, for me that's the difference between `require` and `assert`, like you're comparing a number of fields in the structure or a number of elements in the array, you don't want to stop checking after the first failure. But I also admit that our current codebase tends to use `assert`s more than it should be using. At the same time it's not that big of a problem, the test will fail anyway, it just regulates the amount of garbage produced when it fails.

Reply:
I use `require` for something which should happen or all other tests will have no meaning (e.g. if method returned an error, all other returned values are probably nil).
`assert` for multiple checks which still can be performed even if one of them fail.